### PR TITLE
Chicken Tenders Heal Janitors

### PR DESCRIPTION
**Pull request by west3436**
memes

He does it for ~~free~~ delicious tendies.

:cl:
 * rscadd: Chicken Tenders now heal Janitors similarly to how sprinkles heal Security members.
 * tweak: Changed "Chicken Fillet" to "Chicken Tenders" for lore consistency.

### CODE REVIEW

File: code/modules/reagents/Chemistry-Reagents.dm [hunk 1]
```diff
@@ -5278,6 +5278,28 @@
...
+
+/datum/reagent/tendies/on_mob_life(var/mob/living/M)
+
+	if(..())
+		return 1
+
+	M.nutrition += REM * nutriment_factor
+	if(ishuman(M))
+		var/mob/living/carbon/human/H = M
+		if(H.job == "Janitor")
```
Comment (ShiftyRail) [inline code/modules/reagents/Chemistry-Reagents.dm]: You want that to be `H.mind.assigned_role`.

Comment (west3436) [reply code/modules/reagents/Chemistry-Reagents.dm]: Just copy-pasted sprinkles code for this and adjusted it. I'll replace it with that one if it works better then.

Comment (ShiftyRail) [reply code/modules/reagents/Chemistry-Reagents.dm]: Fairly sure this would mean this effect wouldn't work on cloned Jannies.

**Comment by help-maint** [comment]
Based

File: code/modules/reagents/Chemistry-Reagents.dm (hunk 1 above)
Comment (help-maint) [reply code/modules/reagents/Chemistry-Reagents.dm]: Please update sprinkles to this system too while you're at it

**Comment by Eneocho** [comment]
Kek, the speedmerge.

**Pull request merged**